woocommerce/wc-admin
# Update payment card style on mobile

Fixes #6393 

Update payment card style on mobile

### Screenshots
Before
<img width="528" alt="before" src="https://user-images.githubusercontent.com/56378160/108786769-550ae200-7542-11eb-95e0-579bd66995e1.png">

After
<img width="529" alt="after" src="https://user-images.githubusercontent.com/56378160/108786780-589e6900-7542-11eb-88a0-3800f518a353.png">


### Detailed test instructions:

-   Go to your WooCommerce -> Home -> Choose payment methods.
-   See that card style render as expected.
